composer.json

```diff
@@ -5,7 +5,7 @@
     "keywords": ["toby", "laravel", "hr"],
     "license": "MIT",
     "require": {
-        "php": "^8.3.8",
+        "php": "^8.3.0",
```

Comment:
What is the reason of this change? PHP in the docker image is 8.3.8, and any composer install/update command will failed in this case.

Reply:
fixed, I had failure earlier because I do not have the newest build of the app.. and I made change to 8.3.0 version.